src/map/packet_system.cpp

```diff
@@ -1018,18 +1018,30 @@ void SmallPacket0x028(map_session_data_t* const PSession, CCharEntity* const PCh
     int32 quantity  = data.ref<uint8>(0x04);
     uint8 container = data.ref<uint8>(0x08);
     uint8 slotID    = data.ref<uint8>(0x09);
+    CItem* PItem    = PChar->getStorage(container)->GetItem(slotID);
```

Comment:
`GetItem()` can potentially return `nullptr`. You should check `PItem` exists before you use it:
```cpp
if (PItem == nullptr) { return; }
```